deltachat-ffi/deltachat.h

```diff
@@ -5709,8 +5709,14 @@ int64_t         dc_lot_get_timestamp     (const dc_lot_t* lot);
 #define DC_CERTCK_STRICT 1
 
 /**
- * Accept invalid certificates, including self-signed ones
- * or having incorrect hostname.
+ * Accept certificates that are expired, self-signed
+ * or otherwise not valid for the server hostname.
```

Comment:
```suggestion
 * or not valid for the server hostname.
```
`DC_CERTCK_ACCEPT_INVALID` enables using `async_native_tls` with the `danger_accept_invalid_certs(true)` option which still should reject revoked certificates AFAIU, but it's not clear from [the documentation](https://docs.rs/native-tls/latest/native_tls/struct.TlsConnectorBuilder.html#method.danger_accept_invalid_certs)

Reply:
i just copied the documentation from the rust part :)